faunadb-scala/src/main/scala/faunadb/values/Value.scala

```diff
@@ -102,6 +102,8 @@ object Value {
 
   /** Create a date value. */
   def apply(localdate: LocalDate): Value = DateV(localdate)
+
+  implicit def from[T](obj: T)(implicit encoder: Encoder[T]): Value = encoder.encode(obj)
```

Comment:
I tried to move this function to `Codec` object, but it loses the ability to convert from any type `T` to `Value`. Do I have to do anything different?

```
[error] faunadb-jvm/faunadb-scala/src/test/scala/faunadb/CodecSpec.scala:152: type mismatch;
[error]  found   : CodecSpec.this.Product
[error]  required: faunadb.values.Value
[error]     val product: Value = Product("laptop", 999)
[error]                                 ^
[error] faunadb-jvm/faunadb-scala/src/test/scala/faunadb/CodecSpec.scala:158: type mismatch;
[error]  found   : CodecSpec.this.Order
[error]  required: faunadb.values.Value
[error]     val order: Value = Order("John", Seq(Product("laptop", 999), Product("mouse", 9.99)))
```